pkg/kexec/memory_linux.go

```diff
@@ -10,30 +10,80 @@ import (
 	"fmt"
```

Comment:
I remember we decided a few weeks back to move the memory-related function to its own "memory" package.

Reply:
I'd like to do that in a later CL. I actually am not sure that makes sense anymore -- I originally moved stuff around, but it's easier to review changes like this separately.

Reply:
I.e. the diff is easier to read if everything is still in the same place. We can do a move later.

Reply:
Later pull request, sounds good